src/db/mod.rs

```diff
@@ -28,19 +28,27 @@ pub fn connect_db() -> Result<Connection, failure::Error> {
 pub(crate) fn create_pool() -> r2d2::Pool<r2d2_postgres::PostgresConnectionManager> {
     let db_url = env::var("CRATESFYI_DATABASE_URL")
         .expect("CRATESFYI_DATABASE_URL environment variable is not exists");
-    let pool_size = env::var("DOCSRS_POOL_SIZE")
+    let max_pool_size = env::var("DOCSRS_MAX_POOL_SIZE")
         .map(|s| {
             s.parse::<u32>()
-                .expect("DOCSRS_POOL_SIZE must be an integer")
+                .expect("DOCSRS_MAX_POOL_SIZE must be an integer")
         })
         .unwrap_or(90);
 
+    let min_pool_idle = env::var("DOCSRS_MIN_POOL_IDLE")
+        .map(|s| {
+            s.parse::<u32>()
+                .expect("DOCSRS_MIN_POOL_IDLE must be an integer")
+        })
+        .unwrap_or(10);
+
```

Comment:
Maybe this should be kept an `Option` and then later passed to `.min_idle` as an `Option<u32>` so that prod keeps the same behavior it's been having

Reply:
I don't think having the idle set to the max as a default makes sense, you never really want the connections sitting around idle if they're never used, with the default 10 minute timeout before idle connections expire you essentially have the pool sit at the maximum used in the last 10 minutes + this value extra connections.